app/views/tournament/leaderboard.scala

```diff
@@ -79,11 +79,11 @@ object leaderboard {
           h1("Tournament winners"),
           div(cls := "tournament-leaderboards")(
             eliteWinners,
-            freqWinners(winners.hyperbullet, PerfType.Bullet, "HyperBullet"),
-            freqWinners(winners.bullet, PerfType.Bullet, "Bullet"),
```

Comment:
You also changed the text "Bullet" vs "bullet"

Reply:
Ah yeah good spot!